test/integration/scheduler_perf/scheduler_perf.go

```diff
@@ -1276,7 +1276,7 @@ func runWorkload(tCtx ktesting.TContext, tc *testCase, w *workload, informerFact
 						select {
 						case <-ticker.C:
 							for i := range churnFns {
-								retVals[i][count%concreteOp.Number] = churnFns[i](retVals[i][count%concreteOp.Number])
+								retVals[i][(count+1)%concreteOp.Number] = churnFns[i](retVals[i][count%concreteOp.Number])
```

Comment:
Sorry for missing the message, but can you help to explain why `count+1` here, I thought it should be the same as the parameter for churnFns, then it will transfer like `"" -> "<name>" -> ""`